parameter/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+include (GenerateExportHeader)
```

Comment:
Please put it right before its usage, so that:

- it's obvious what it is for
- we won't forget to remove it in case we don't use it anymore.